images/base/Dockerfile

```diff
@@ -124,8 +131,17 @@ RUN echo "Ensuring scripts are executable ..." \
     && sha256sum --ignore-missing -c /tmp/runc.sha256 \
     && mv /tmp/runc.${TARGETARCH} /usr/local/sbin/runc \
     && chmod 755 /usr/local/sbin/runc \
+    && curl -sSL --retry 5 --output /tmp/stargz-snapshotter.${TARGETARCH}.tgz "${STARGZ_SNAPSHOTTER_URL}" \
+    && echo "${STARGZ_SNAPSHOTTER_AMD64_SHA256SUM}  /tmp/stargz-snapshotter.amd64.tgz" | tee /tmp/stargz-snapshotter.sha256 \
+    && echo "${STARGZ_SNAPSHOTTER_ARM64_SHA256SUM}  /tmp/stargz-snapshotter.arm64.tgz" | tee -a /tmp/stargz-snapshotter.sha256 \
+    && echo "${STARGZ_SNAPSHOTTER_PPC64LE_SHA256SUM}  /tmp/stargz-snapshotter.ppc64le.tgz" | tee -a /tmp/stargz-snapshotter.sha256 \
+    && sha256sum --ignore-missing -c /tmp/stargz-snapshotter.sha256 \
+    && rm -f /tmp/stargz-snapshotter.sha256 \
+    && tar -C /usr/local/bin/ -xzvf /tmp/stargz-snapshotter.${TARGETARCH}.tgz \
+    && rm -f /usr/local/bin/ctr-remote \
     && containerd --version \
     && runc --version \
+    && containerd-stargz-grpc --help \
```

Comment:
would be nice to have --version added to this CLI eventually.

Reply:
Started to track this at https://github.com/containerd/stargz-snapshotter/issues/268.